sklearn/naive_bayes.py

```diff
@@ -386,16 +424,9 @@ def _partial_fit(self, X, y, classes=None, _refit=False,
 
         classes = self.classes_
 
-        unique_y = np.unique(y)
-        unique_y_in_classes = np.in1d(unique_y, classes)
-
-        if not np.all(unique_y_in_classes):
-            raise ValueError("The target label(s) %s in y do not exist in the "
-                             "initial classes %s" %
-                             (unique_y[~unique_y_in_classes], classes))
-
-        for y_i in unique_y:
-            i = classes.searchsorted(y_i)
+        for y_i in np.unique(y):
+            # classes need not be sorted because user input
+            i = np.where(classes == y_i)[0][0]
```

Comment:
i'll sort them but then i'll also make a note in the documentation of this parameter that this will be sorted internally and they should use the classes_ argument to get the final order. works?